### PR TITLE
feat/Add sortable functionality

### PR DESCRIPTION
**Pull request by gassius**
This PR adds a sortable functionality to the package. It allows setting the `sortable` property for header's `Cell` objects to allow the sorting of a column. (Examples added to the README.md). It uses the `SortableIcon` Vue component from Laravel Nova (available as global Vue component) and the `InteractsWithQueryString` mixing from the [laravel-nova](https://www.npmjs.com/package/laravel-nova) package to keep the same UX/UI of the main package.

Also included the optional `sortableData` property to the `Cell` objects that allows to define a different value for display and sorting.

### CODE REVIEW

**Comment by gassius** [comment]
@m-a-k-o any feedback?

File: resources/js/components/Card.vue [hunk 1]
```diff
@@ -38,13 +47,78 @@
...
+import { InteractsWithQueryString } from 'laravel-nova'
+
 export default {
+    mixins: [InteractsWithQueryString],
     props: ['card'],
 
+    /**
+     * Mount the component and retrieve its initial data.
+     */
+    async created() {
```
Comment (m-a-k-o) [inline resources/js/components/Card.vue:60]: async is not required here.

**Review by m-a-k-o — APPROVED** [review]

**Comment by m-a-k-o** [comment]
@gassius Sorry for late feedback.

Merged and released as https://github.com/m-a-k-o/nova-custom-table-card/releases/tag/3.3.0.

**Comment by m-a-k-o** [comment]
Thank you for your contribution 👌

**Pull request merged**